includes/Lister/Lister.php

```diff
@@ -7,6 +7,8 @@
 use MediaWiki\Extension\DynamicPageList3\Parameters;
 use MediaWiki\Extension\DynamicPageList3\UpdateArticle;
 use MediaWiki\MediaWikiServices;
+use PageImages\PageImages;
+use ExtensionRegistry;
```

Comment:
Move this above for alphabetical order.

Reply:
Also doesn't look like PageImages\PageImages is used so can be removed

Reply:
Done